README.md

```diff
@@ -26,11 +26,20 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
 default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
 ```
 
+If you wish to use custom SSL certificates, define the following `config` attributes:
+
+```ruby
+default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/ssl.crt'
+default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/ssl.key'
+```
+
+To find out all supermarket `config` attributes you can override, see [omnibus-supermarket](https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb). An attribute like `default['supermarket']['nginx']['log_rotation']['num_to_keep'] = 10` translates to  `supermarket-omnibus-cookbook` as `default['supermarket_omnibus']['config']['nginx']['log_rotation']['num_to_keep'] = 10`
```

Comment:
Being new to the wrapper of wrappers, it took me a little while to trace this. What do you think of a little expansion of this portion of the README?:

--

To find out all supermarket `config` attributes you can override, see [omnibus-supermarket](https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb). Translation of attributes from `supermarket-omnibus-cookbook` to attributes in `omnibus-supermarket` occurs in the `supermarket_server` resource provided by this cookbook which produces a JSON that `omnibus-supermarket` reads.

```ruby
# an attribute you define via this supermarket_omnibus_cookbook
default['supermarket_omnibus']['config']['nginx']['log_rotation']['num_to_keep'] = 10

# becomes the following in omnibus_supermarket
default['supermarket']['nginx']['log_rotation']['num_to_keep'] = 10
```